components/Blocks/PlainHero.css

```diff
@@ -16,7 +16,7 @@
   margin: 0 auto 0.5em;
   font-weight: 100;
   line-height: 1.1;
-  width: 31rem;
+  max-width: 31rem;
```

Comment:
Yeah, just wary that the max-width _will_ increase if the base font size. It’s not necessarily a bad thing though, we should just explicitly agree that’s what we’re going for. Documented assumptions innit m8.

But yeah, feature, not a bug, merge.